src/customize/components/Customize.js

```diff
@@ -74,19 +74,18 @@ export default class Customize extends React.Component {
       }
       return (
         <div key={index}>
-          <h5>{option.label}</h5>
           {input}
         </div>
       );
     });
     return (
       <div className="container">
-        <h2>2. Customize</h2>
+        <h2 classsName="form-group__label">2. Customize</h2>
```

Comment:
This is not a form group, only a header so this class should be removed. All though IMO we can remove the header completely.